packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.css

```diff
@@ -17,3 +17,56 @@ li {
 .facet__value-checkbox {
   min-width: unset;
 }
+
+.facet__value_new-checkbox{
+  width: 20px;
+  height: 20px;
+  vertical-align: top;
+  border: solid 1px #cccccc;
+  border-radius: 4px;
+  margin-right: 6px;
+  display: inline-block;
+  position: relative;
+
+}
+
+
+
+.facet__value_checkbox-checked{
+  color: #fff;
+  position: absolute;
+  top: 3px;
+  left: 2px;
+  width: 13px;
+  height: 13px;
+}
+.facet__value_checkbox-checked::before{
+  width: 6px;
+  border-top-left-radius: 1px;
+  border-bottom-left-radius: 1px;
+  top: 5px;
+  left: 1px;
+  transform: rotate(49deg);
+
+  content: '';
+  height: 2px;
+  background-color: #0070d2;
+  display: block;
+  position: absolute;
+  transform-origin: left center;
+
+}
+.facet__value_checkbox-checked::after{
+  content: '';
+  height: 2px;
+  background-color: #0070d2;
+  display: block;
+  position: absolute;
+  width: 13px;
+  border-top-right-radius: 1px;
+  border-bottom-right-radius: 1px;
+  top: 10px;
+  left: 4px;
+  transform: rotate(-49deg);
+  transform-origin: left center;
+}
```

Comment:
Looks like this isn't used anywhere

Reply:
> 

Left over from another implementation. Should be removed/